package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bigcommerce/checkout",
-  "version": "1.110.3",
+  "version": "1.110.0",
```

Comment:
It should not change the version on the module, it is managed by circle-ci

Reply:
fixed